app/policies/collection_policy.rb

```diff
@@ -27,10 +25,10 @@ def update?
     administrator? || manages_collection?(record)
   end
 
-  # Can works be deposited into this collection
+  # Can this collection be deposited
   sig { returns(T::Boolean) }
   def deposit?
-    NO_DEPOSIT_STATUS.exclude? record.state
+    record.can_begin_deposit?
```

Comment:
A magic method, I assume. Where's it from? What's it doing?

Reply:
It's from the state-machine gem. It checks to see if it's in a state where it could do the begin_deposit transition.